docs/logs.md

```diff
@@ -0,0 +1,67 @@
+# Cachi2 Logging Standards
+
+## Logging principles
+
+1) Use a Consistent Format: To ensure that logs are easy to read and understand, use a consistent format throughout the system. This format should include the date and time of the log entry, the severity level of the log message, and the message itself.
+2) Use Descriptive Severity Levels: Use descriptive severity levels to indicate the importance of a log message. For example, use "INFO" for informational messages, "WARN" for problems that do not interfere with the final result of the request, and "ERROR" for errors that affect the final result of the request.
```

Comment:
Copying Bruno's points from the original design doc

> Should we also detail here the use of DEBUG?

> Maybe we draw inspiration from Syft.
> 
> The INFO level output is very lean, and it just outlines the overall steps Syft is taking:
> <https://pastebin.com/wBuGj5u5>
> 
> The DEBUG level is very detailed:
> <https://pastebin.com/3Pw9dfwc>

Reply:
DEBUG is interesting. Currently, we don't enable debug logs by default (not even in rhtap) so any info logged at debug level is as good as lost.

Reply:
We could make debug level the default in RHTAP. I think the question here is more of "how verbose we want INFO to be", since we're setting guidelines.

Do we want to go for a default clean approach, or a default verbose approach? I think it's fine if we decide going towards the latter, but then we'd need to set specific guidelines on what should be DEBUG (if anything at all).

Reply:
I think, more realistically, we should identify the most important events to log. Some of those might be:

* Starting to process a package
* Resolving the package name and version
  * And the project files parsed / commands used to do that
* Identifying dependencies
  * And the lockfiles parsed / commands used to do that
* Validating dependencies
  * maybe also what validation is being performed?
* Downloading dependencies